### PR TITLE
Remove TopUp expiration

### PR DESCRIPTION
**Pull request by ihor-palii**
Disable expiring credits email alert
https://app.asana.com/0/1159876841343611/1200489923553644/f

Remove credits expiration date
https://app.asana.com/0/1159876841343611/1200522300500824/f

### CODE REVIEW

File: temba/orgs/migrations/0074_remove_topup_expires_on.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        migrations.AlterField(
+            model_name="topup",
+            name="expires_on",
+            field=models.DateTimeField(
+                default=year_from_now,
+                help_text="The date that this top up will expire",
+                verbose_name="Expiration Date",
+            ),
+        ),
+        migrations.RemoveField(
```
Comment (teehamaral) [inline temba/orgs/migrations/0074_remove_topup_expires_on.py:26]: This field is also used on Mailroom (https://github.com/nyaruka/mailroom/blob/72657f3f3a146028f6b0955cea08d313c462f028/core/models/topups.go#L127), so I think we should keep it on the code to make it simple on the next merging processes. So I think we only should not include it on the calc of the date expiration.

File: temba/orgs/models.py [hunk 2]
```diff
@@ -2596,21 +2566,17 @@ class TopUp(SmartModel):
...
-    def create(cls, user, price, credits, stripe_charge=None, org=None, expires_on=None, comment=None):
+    def create(cls, user, price, credits, stripe_charge=None, org=None, comment=None):
         """
         Creates a new topup
         """
         if not org:
             org = user.get_org()
 
-        if not expires_on:
-            expires_on = timezone.now() + timedelta(days=365)  # credits last 1 year
```
Comment (teehamaral) [inline temba/orgs/models.py:2607]: Let's make the default expiration time something like 50 years

File: temba/orgs/migrations/0074_remove_topup_expires_on.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.2.10 on 2021-06-28 13:33
+
+from django.db import migrations, models
+from django.utils.timezone import now as tz_now, timedelta
+
+
+def year_from_now():
+    return tz_now() + timedelta(days=365)
```
Comment (teehamaral) [inline temba/orgs/migrations/0074_remove_topup_expires_on.py:8]: Let's make a migration to update the expiration time for something like 50 years from now

**Review by teehamaral — CHANGES_REQUESTED** [review]

File: temba/orgs/migrations/0074_auto_20210630_1156.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("orgs", "0073_merge_20201214_1542"),
+    ]
+
+    operations = [
+        migrations.RunPython(update_topup_expiration_to_last_50_years, update_topup_expiration_to_last_1_year)
```
Comment (teehamaral) [inline temba/orgs/migrations/0074_auto_20210630_1156.py:26]: Quick question: should we have both `update_topup_expiration_to_last_50_years` and `update_topup_expiration_to_last_1_year` here?

**Review by teehamaral — CHANGES_REQUESTED** [review]

File: temba/orgs/migrations/0074_auto_20210630_1156.py (hunk 4 above)
Comment (ihor-palii) [reply temba/orgs/migrations/0074_auto_20210630_1156.py:26]: `update_topup_expiration_to_last_1_year ` it's method that will be executed on migration revert. We can't revert it to the previous value but able to decrease it to one year as we had in the previous version. I think it's a good practice to provide a reverse method too.

Comment (teehamaral) [reply temba/orgs/migrations/0074_auto_20210630_1156.py:26]: Oh, got it. Makes sense.

**Review by teehamaral — APPROVED** [review]

**Pull request merged**